ci/ansible/roles/pulp-priority/tasks/main.yml

```diff
@@ -3,3 +3,39 @@
   package:
     name: yum-plugin-priorities
     state: present
+
+- name: Setup Pulp nightly repository
```

Comment:
@bherrin3, as we talked let`s look for a solution that do not require duplicate the creation of those repos.

Reply:
@kersommoura updated to be more dry once you let me know about `pre_tasks` ! Neat!

See if you prefer this implementation.

I need to check to see if `pulp-priority` role is being used by anything else. If not, I can remove that file in this PR as well.